src/main/scala/org/scalajs/tools/tsimporter/sc/Printer.scala

```diff
@@ -113,10 +113,12 @@ class Printer(private val output: PrintWriter, outputPackage: String) {
       case sym: ModuleSymbol =>
         pln"";
         pln"@js.native"
-        if (currentJSNamespace.isEmpty)
-          pln"@JSGlobal"
-        else
-          pln"""@JSGlobal("$currentJSNamespace${name.name}")"""
+        if (sym.isGlobal) {
+          if (currentJSNamespace.isEmpty)
+            pln"@JSGlobal"
+          else
+            pln"""@JSGlobal("$currentJSNamespace${name.name}")"""
+        }
         pln"object $name extends js.Object {"
```

Comment:
Non-global objects should also be non-`@js.native` and not `extends js.Object`. Because they don't actually exist in the JavaScript code. They are only a Scala-side construction.

Reply:
fixed at https://github.com/sjrd/scala-js-ts-importer/pull/57/commits/021cca003a05b1f2df8eaefb3260d4d454e61e43